perimysium/dataman.py

```diff
@@ -409,6 +409,18 @@ def write(self, fpath):
 
         f.close()
 
+    def addnoise(self, noiseWidth):
```

Comment:
Can you rename this as `add_noise`? See the python style guidelines: https://www.python.org/dev/peps/pep-0008/#function-names